### ISSUE TITLE
local test network file

### ISSUE THREAD

**Issue by pedro-sys** [AshAvalanche/ansible-avalanche-getting-started#36]
Is missing the network ID of the X chain
I got this error with the command "ash avalanche subnet list"
Error loading network: Config error: failed to deserialize configuration from 'terraform/multipass/local-test-network.yml': missing field `id`

**Comment by leopaul36** [comment]
Hi @pedro-sys, thanks for reporting this.

What version of Ash CLI are you running? (`ash -V`). You are probably running an older version.

Please upgrade to [v0.2.2](https://github.com/AshAvalanche/ash-rs/releases/tag/v0.2.2) and let us know if you still have an issue.

Here's what the command looks like on my side:

```
ash -c terraform/multipass/local-test-network.yml avalanche subnet list --network local

Found 5 Subnet(s) on network 'local':
-------------------------------------------------------
- 29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL:
  Type: Permissioned
  Control keys: ["P-local18jma8ppw3nhx5r4ap8clazz0dps7rv5u00z96u"]
  Threshold:    1
  Blockchains list (1):
  - AshLocalEVM:
    ID:      RfX8YRUHePxFENBZFwaNt1tP6CR6RQ4hJJ5AqYv49LRoSKniK
    VM ID:   srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
    VM type: SubnetEVM
------------------------------------------
- 11111111111111111111111111111111LpoYY:
  Type: PrimaryNetwork
  Blockchains list (3):
  - P-Chain:
    ID:      11111111111111111111111111111111LpoYY
    VM ID:   11111111111111111111111111111111LpoYY
    VM type: PlatformVM
    RPC URL: http://10.252.190.97:9650/ext/bc/P
  - C-Chain:
    ID:      2CA6j5zYzasynPsFeNoqWkmTCt3VScMvXUZHbfDJ8k3oGzAPtU
    VM ID:   mgj786NP7uDwBCcq6YwThhaN8FLyybkCa4zBWTQbNgmK6k9A6
    VM type: SubnetEVM
  - X-Chain:
    ID:      2eNy1mUFdmaxXNj1eQHUe7Np4gju9sJsEtWQ4MX3ToiNKuADed
    VM ID:   jvYyfQTxGMJLuGWa55kdP2p2zSUYsQ5Raupu4TW34ZAUBAbtq
    VM type: AvalancheVM
    RPC URL: http://10.252.190.97:9650/ext/bc/X
```

**Comment by pedro-sys** [comment]
ash_cli 0.2.0
Ok I'll try

**Comment by Nuttymoon** [comment]
Hi @pedro-sys, I just released `v0.2.3` both on GitHub and on https://crates.io. You should be able to install it whatever your preferred method!

**Comment by pedro-sys** [comment]
Just installed v0.2.3 :) 
works fine !!
